gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/accessController/index.ts

```diff
@@ -10,7 +11,7 @@ export default [
   Initialize,
   AddAccess,
   ReadState,
-  makeAcceptOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER),
-  makeTransferOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER),
+  makeAcceptOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER, ReadOwnership),
+  makeTransferOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER, ReadOwnership),
```

Comment:
Let's have a generic command for inspecting the ownership:
```suggestion
  makeAcceptOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER),
  makeTransferOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER),
  makeInspectOwnershipCommand(CONTRACT_LIST.ACCESS_CONTROLLER),
```

Reply:
Just noticed that Kyle is working on it already, you can leave it for him then: https://github.com/smartcontractkit/chainlink-solana/pull/273/files#diff-c23d133098f9cc688d9a4467a87a83fd63202c4134649e6e868b9a30c55611ef